src/com/lftechnology/java/training/alina/basic/stemleaf/StemLeafRepresentation.java

```diff
@@ -0,0 +1,90 @@
+package com.lftechnology.java.training.alina.basic.stemleaf;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.InputMismatchException;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Java program to present data in stem and leaf form
+ * 
+ * @author Alina Shakya <alinashakya@lftechnology.com>
+ */
+public class StemLeafRepresentation {
+
+	public static final int MIN_VALUE = 3;
+	public static final int MAX_VALUE = 99;
+	private static final Logger LOGGER = Logger
+			.getLogger(StemLeafRepresentation.class.getName());
+	private static int dataPoints;
+	private static DataSet[] data;
+
+	public static void main(String[] args) {
+		try (Scanner scanner = new Scanner(System.in)) {
+			getDataPoints(scanner);
+			getStemLeaf();
+		} catch (InputMismatchException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)